db/migrate/20190814064811_change_column_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumnName < ActiveRecord::Migration[5.2]
```

Comment:
you can just run `rake db:rollback` and then rename this column to not push a couple of migrations

Reply:
But if we already have information in the database?

Reply:
@rbUUbr they use one database for all things)